articles/industry-4.0-and-cybersecurity/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/industry-4.0-and-cybersecurity/
+title: Industry 4.0 and Cybersecurity
+description: An introduction to Industry 4.0 through the enabling technologies, including fundamental principles of security, cybersecurity challenges, threats and insinuations.
+author: collins-agesa
+date: 2020-09-06T00:00:00-08:00
+topics: [Cybersecurity]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/industry-4.0-and-cybersecurity/hero.jpg
+    alt: bulma image example
+---
+Many technologies have emerged in recent times. These technologies include the Internet of Things, Big Data Analytics and Cyber-Physical Systems. They have enabled new and impactful initiatives such as Industry 4.0. this initiative is revolutionizing the manufacturing industry. Yet, the cybersecurity challenges associated with the technology have never been greater.
+<!--more-->
+
+## Industry 4.0
+
+Industry 4.0 had been a concept of German origin as a government initiative.  Dubbed as the fourth industrial revolution, it was first introduced as Industrie 4.0 in 2011. The general goal was to improve competitiveness of the manufacturing industry. It was also to advance the new generation of smart manufacturing systems. Enabling technologies played a big part to support this goal. These technologies include:
+
+***Internet of Things (IoT):** a network of interconnected objects. Objects are physical devices and are billions in number globally, through the internet. IIoT refers to Industrial Internet of Things.
+
+***Cyber-Physical Systems:** integrations of physical processes with computation and networking processes. Key for automation and monitoring.
```

Comment:
This entire section on technologies features very basic definitions with little to no explanations of what they are/do and _why_ exactly they are relevant to Industry 4.0. Make sure to bring it back or relate it explicitly to your thesis.